stable/lamp/README.md

```diff
@@ -3,6 +3,8 @@
 As of Nov 13, 2020, charts in this repo will no longer be updated.
 For more information, see the Helm Charts [Deprecation and Archive Notice](https://github.com/helm/charts#%EF%B8%8F-deprecation-and-archive-notice), and [Update](https://helm.sh/blog/charts-repo-deprecation/).
 
+This chart continuous to be maintained at [https://github.com/lead4good/lamp-helm-repository](https://github.com/lead4good/lamp-helm-repository).
```

Comment:
Should probably be "continues" not "continuous".

Reply:
Sorry for the typo, issue has been adressed